examples/NeuralNetwork-color-classifier/sketch.js

```diff
@@ -1,3 +1,8 @@
+// Copyright (c) 2023 ml5
```

Comment:
This is maybe a silly comment, but I sometimes find that when looking at the examples that the copyright notice at the top is a little off-putting / intimidating. Being clear and transparent about software licenses is important, however, maybe we should try a friendlier heading? 

Also, the ml5 project has its own license, not MIT! What if we create a new uniform heading something like:

// 👋 Hello! This is an ml5.js made and shared with ❤️.
// Learn more about the ml5.js project: https://ml5js.org/
// ml5.js license and code of conduct: https://github.com/ml5js/ml5-next-gen/blob/main/LICENSE.md

// This example demonstrates _______________
// Any other notes?

Reply:
Tagging in @sproutleaf and @acosmica in case they have any thoughts!

Reply:
Agree with your sentiment Dan! I think a new uniform heading with some helpful links would be great.

Should I open an issue to track the progress of heading revamp?